src/main/java/thePackmaster/powers/odditiespack/FinalFormPower.java

```diff
@@ -43,14 +45,13 @@ public void update() {
                 att(new NewQueueCardAction(q, true, true, true));
             }
         });
-
-
+        atb(new ReducePowerAction(owner, owner, this, 1));
     }
 
     @Override
     public void updateDescription() {
         if (card != null)
-            description = DESCRIPTIONS[0] + FontHelper.colorString(card.name, "y") + DESCRIPTIONS[1];
+            description = DESCRIPTIONS[0] + amount + (amount == 1 ? DESCRIPTIONS[1] : DESCRIPTIONS[2]) + DESCRIPTIONS[3] + FontHelper.colorString(card.name, "y") + DESCRIPTIONS[4];
```

Comment:
make sure that zhs has these

Reply:
Just adding the ENG version of the strings now for crash safety, I don't want to step on mwalls' work